src/testing/flow_solver.cpp

```diff
@@ -27,129 +7,61 @@ namespace Tests {
 // FLOW SOLVER TEST CASE -- What runs the test
 //=========================================================
 template <int dim, int nstate>
-FlowSolver<dim, nstate>::FlowSolver(const PHiLiP::Parameters::AllParameters *const parameters_input)
+FlowSolver<dim, nstate>::FlowSolver(const PHiLiP::Parameters::AllParameters *const parameters_input, std::shared_ptr<FlowSolverCaseBase<dim, nstate>> flow_solver_case)
 : TestsBase::TestsBase(parameters_input)
+, flow_solver_case(flow_solver_case)
 , initial_condition_function(InitialConditionFactory<dim,double>::create_InitialConditionFunction(parameters_input, nstate))
-, all_param(*(TestsBase::all_parameters))
+, all_param(*parameters_input)
 , flow_solver_param(all_param.flow_solver_param)
 , ode_param(all_param.ode_solver_param)
-, courant_friedrich_lewy_number(flow_solver_param.courant_friedrich_lewy_number)
 , poly_degree(all_param.grid_refinement_study_param.poly_degree)
 , final_time(flow_solver_param.final_time)
 , unsteady_data_table_filename_with_extension(flow_solver_param.unsteady_data_table_filename+".txt")
```

Comment:
This should be moved to `periodic_cube_flow`

Reply:
Fixed in recent commit